packages/checkout/widgets-lib/src/widgets/sale/views/PayWithCard.tsx

```diff
@@ -64,8 +64,8 @@ export function PayWithCard() {
       transactionId: signData?.transactionId,
     };
 
-    sendSuccessEvent(SaleWidgetViews.SALE_SUCCESS, [], signTokenIds, details);
-    sendCloseEvent(SaleWidgetViews.SALE_SUCCESS);
+    sendSuccessEvent(SaleWidgetViews.SALE_SUCCESS, [], signTokenIds, details); // checkoutPrimarySaleSaleSuccess_SuccessEventSucceeded
```

Comment:
do we need this comments?

Reply:
The comments are there to make Looker events string searchable.